clojure/README.md

```diff
@@ -0,0 +1,49 @@
+# clojure
+
+Clojure is a robust, practical, and fast programming language with a set of useful features that together form a simple, coherent, and powerful tool.
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Service package
```

Comment:
Hi @johnjelinek - 
I was looking at this new plan once more and am a little uncertain whether it should be strictly a Binary package (and require people who need the NREPL running to create their own service plan) or if there's a fundamental need to include the NREPL as a service? 
I know a little Clojure but would appreciate a little education here.

Apologies for flip-flopping on approving this PR but changing the package type after merging this PR would be a breaking change.

Reply:
I'm wrong, this is not a breaking change - I can change this myself after this merges.